lab2.js

```diff
@@ -48,7 +48,21 @@ function assert(expression, failureMessage) {
  with Dowington.
 */
 
-var hoursSpentInDowington; // TODO: assign me the value of the
+function Blob() {}
+var blob = new Blob();
+
+var consumptionRate = 1;
+var hours = 0;
+var persons = 1000;
+var startingPersons = 1000;
+while (persons > 0) {
+  hours++;
+  persons -= consumptionRate;
+  consumptionRate = (consumptionRate + (-persons + startingPersons)); //consumption rate is a fibbonaci sequence :)
```

Comment:
:) yes, that's one way to do it. I realize I wasn't super-specific on the rate implementation, so this works. And it's fun.